__tests__/app.test.js

```diff
@@ -32,3 +32,24 @@ describe(' GET /api/topics', () => {
     });
 })
 
+describe('GET /api/articles/:article_id', () => {
+    test('status:200, responds with a single matching article', () => {
+        const article_id = 1
+        return request(app)
+        .get(`/api/articles/${article_id}`)
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body;
+            expect(article).toBeInstanceOf(Object)
```

Comment:
you could remove this assertion as we are implicitly checking this in the next assertion